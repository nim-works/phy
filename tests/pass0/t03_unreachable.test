discard """
  output: "(Error ecUnreachable)"
"""
(TypeDefs
  (ProcTy (Void))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 0) 0 (Locals) (List
    (Block (Params)
      (Unreachable)
    )
  ))
)