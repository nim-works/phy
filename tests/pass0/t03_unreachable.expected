.type t0 () -> void
.start t0 p0
  Unreachable
.end
